src/app/components/list/view-switcher/index.html

```diff
@@ -21,44 +21,44 @@
       <h3 class="sky-text-success">Use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Use the list view switcher when the user needs to switch between different views of the same data.</p>
+              <p>Use list view switchers when users need to switch between different views of the same data.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/use-when.png" />
-              <div class="caption caption-success sky-text-success">Do use the list view switcher when there is more than one view of the data.</div>
+              <div class="caption caption-success sky-text-success">Do use list view switchers when more than one view of data is available to users.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
-      <h3 class="sky-text-success">Don't use when</h3>
+      <h3 class="sky-text-danger">Don't use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Don't use the list view switcher when there is only one view of the data.</p>
+              <p>Don't use list view switchers when only one view of data is available.</p>
```

Comment:
@Blackbaud-ToddRoberts, is this Don’t use when item really necessary? It’s probably better to be explicit than to assume that users can/will read between the lines, but this item seems implicit based on the Use when item, so just making sure that we actually need it.

Reply:
Probably extraneous but @Blackbaud-AdamFunderburk and I have been discussion whether we should always have "don't" examples - what do you think, Adam?

Reply:
I'm a believer in having at least one example of how a component could plausibly be misapplied. It's not _bad_ to include this in my opinion - I don't think it's that implausible (based on my experience) for a consumer to mistakenly assume that it's a standard part of a list toolbar and should always be included, OR that it's ok to include it in a V1 release of a list feature because later versions will include additional views.

Just while I'm thinking about it, another "don't" example could be using the list view switcher to toggle between different "views" of a list, outside of the presentation options intended. A consumer could mistakenly use this (because it looks cool!) to accommodate different pre-built sort/filter options of the same list, when we would instead recommend using tabs, persistent inline filters, etc.

Reply:
Makes sense to me to err on the side of including at least one "don't" example. ... Do you guys want to add Adam's additional "don't" example? Or just leave it as the existing example?

Reply:
i am inclined to leave it with the existing example

Reply:
Sounds good. Thanks!